test/stat.ts

```diff
@@ -2,16 +2,42 @@ import assert from 'assert';
 import { fixtures } from './helpers/fixture.js';
 
 describe('stat', () => {
-    let repo;
-    before(async () => (repo = await fixtures.base.repo));
-
-    const { data } = fixtures.base;
-
     describe('stat()', () => {
         it('without params', async () => {
+            const repo = await fixtures.base.repo();
             const actual = await repo.stat();
 
-            assert.deepStrictEqual(actual, data.stat);
+            assert.deepStrictEqual(actual, fixtures.base.data.stat);
+        });
+
+        it('counts objects in cruft packs', async () => {
```

Comment:
A case for `fixtures.cruft.repo()` with no options is needed here to test a default value.